traffic_stops/templates/base.html

```diff
@@ -9,10 +9,23 @@
     <meta name="author" content="{% block meta-author %}{% endblock %}">
     <meta name="keywords" content="{% block meta-keywords %}{% endblock %}">
     {% block extra-meta %}{% endblock %}
-    <link href="https://fonts.googleapis.com/css2?family=Bebas+Neue&family=Montserrat:wght@300;400;700&display=swap" rel="stylesheet">
```

Comment:
Is this font unused now?

Reply:
No, just moved it. The diff is misleading.